src/components/widget/WidgetRecommendedBy.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <div class="widget widget--recommended-by">
+    <div class="widget__header d-flex align-items-center" :class="{ 'card-body': widget.card }">
+      <fa icon="users" fixed-width class="mr-2" />
+      <h3 class="m-0 p-0 h5">Latest recommended documents</h3>
```

Comment:
I would add an i18n key for translation to "Latest recommended documents"

Reply:
Ah yes, you're right I forgot.